ci/playbooks/set_crc_insecure_registry.yml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
I f we can consume openshift_setup having the ability to pass to it insecure registries will be nice :)